crates/bevy_window/src/cursor.rs

```diff
@@ -1,39 +1,79 @@
 /// The icon to display for a window's cursor
+/// Examples of all of these cursors can be found [here](https://www.w3schools.com/cssref/playit.asp?filename=playcss_cursor)
+/// This `enum` is simply a copy of a similar `enum` found in `winit`. 
+/// `winit`, in turn, simply used cursor types avilable in the browser.
 #[derive(Debug, Hash, PartialEq, Eq, Clone, Copy)]
 pub enum CursorIcon {
+    /// The platform-dependent default cursor.
     Default,
+    /// A simple crosshair.   
     Crosshair,
+    /// A hand (often used to indicate links in web browsers).    
     Hand,
+    /// Self explanatory.    
```

Comment:
It's obvious, but not all readers have the same background.  What sort of arrow is it, and when is it used,?